x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/schema.ts

```diff
@@ -167,4 +168,28 @@ export const cspmUsageSchema: MakeSchemaFrom<CspmUsage> = {
       alerts_acknowledged_count: { type: 'long' },
     },
   },
+  cloud_account_stats: {
+    type: 'array',
+    items: {
+      account_id: { type: 'keyword' },
+      cloud_provider: { type: 'keyword' },
```

Comment:
not all kspm accounts contained on a cloud_provider, what the value gonna be in that case here?

Reply:
The value will be null if there isn't a cloud_provider.

Reply:
so shouldn't we move that to `cspm` entry?
I saw you did that convention for `kspm`

Reply:
KSPM can `eks` or `aks`  or `gke` which also which is service belonging to a  `cloud_provider`.